server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  attributes: {
+    id: {
+      type: 'string',
+      columnName: '_id'
+    },
+    createdAt: {
+      type: 'number',
+      autoCreatedAt: true,
+    },
+    updatedAt: {
+      type: 'number',
+      autoUpdatedAt: true,
+    },
+    fullName: {
+      type: 'string',
+      required: true
+    },
+    email: {
+      type: 'string',
+      required: true,
+      unique: true,
+      isEmail: true
+    },
+    campus: {
+      type: 'string',
+      required: true
+    },
+    lifestage: {
+      type: 'string',
+      required: true
+    },
+    isMember: {
+      type: 'ref',
+      defaultsTo: false
+    },
+    isBaptised: {
+      type: 'boolean',
+      defaultsTo: false
+    },
+    phoneNumber: {
+      type: 'number',
+      required: true
+    },
+    financeID: {
+      type: 'number'
```

Comment:
correct me if i'm wrong, but based on my understanding from what Joanna explained, a member A can have multiple id's in tithely giving database. If so, i think this shouldn't be a number type

Reply:
will change it to array of strings then